docker-storage-setup.sh

```diff
@@ -578,6 +578,11 @@ EOF
     if ! wait_for_dev ${dev}1; then
       Fatal "Partition device ${dev}1 is not available"
     fi
+    # Sometimes there is still a old filesystem signature where we
+    # have just created our new partition.  Pvcreate will find it and
+    # ask for confirmation to overwrite it.  So we wipe the new
+    # partition first.
+    wipefs -a ${dev}1
```

Comment:
I don't think we should be wiping signatures by default. If there is data, one can still delete the partition just created and recover the data. 

So I like the idea of bailing out if disk has fs signatures present and user can run wipefs before passing that disk to dss.

Alternatively, we can implement a config option for dss, say WIPE_SIGNATURES_DISK and if this is set to true, we will always first wipe all signatures on disk before further processing. It will be off by default.